api/grpc/mpi/v1/file.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+package f5.nginx.agent.api.grpc.mpi.v1.file;
+
+import "common.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/nginx/agent/v3/api/grpc/mpi/v1";
+
+service FileService {
+    // Get the overview of files for a particular configuration version of an instance
+    rpc GetOverview(ConfigVersion) returns (FileOverview) {}
```

Comment:
nit, i think would be nice to document the direction. While conventionally, we know the "Get" is from perspective of the Agent, it'll be good to be explicit.

Reply:
followed the reference here: https://protobuf.dev/programming-guides/style/#services

Reply:
not sure how your link apply here?